### DISCUSSION TITLE
Is this dreambooth really up to 1024?

### DISCUSSION THREAD

**Discussion by leonnn1** [TheLastBen/fast-stable-diffusion#1059] [Q&A] [answered]
Is this value correct?

I was trying to train my models and I made it using 1024x1024 images, but when I looked at the code I saw this, the configuration of the file says that the resolution is 512, I don't get it, is this dreambooth using 1024 or it's cropping the images down to 512?

![image](https://user-images.githubusercontent.com/85142472/208226438-09709bcc-6582-4609-b5bc-0d6307984590.png)

**Comment by TheLastBen** [comment, marked as answer]
no need to use high resolution pictures to train the text encoder.
-your learning rate is too low

**Reply by leonnn1** [reply]
Can you give a dumbed down explaination about it, I am trying to train this every day but I am still a bit overwelmed, like the Unet is to make the model learn the instance images right, but the text encoder and the text encoder concept training makes me a bit confuse.
And about the learning rate, I wanted to have more detailes on the model I previously trained, so I decreased the learning rate to get finer details, I think this is the way to go right?

**Reply by nawnie** [reply]
use the interrogate image option in extras on the web ui of ay pic, when the text encoder trains it pairs that info with the images you are showing it.

set your lr back to 2e-6
use the drop down menu to change resolution don't edit the code until you trace back what all the functions do. dreambooth models are not really interchangeable and what 1 does another may not or use a different syntax to do so.

changing resolution for training the text encoder is pointless due to ... a shirt looks like a shirt at 512 or 4k

**Reply by leonnn1** [reply]
My learning rate was fine, it did what I wanted it to do, this specific part wasn't an issue since I wanted fine detailes this wasn't a first model, it was a custom model that I previously trained that I wanted to add more without changing the shape of things you know? I think that this is the way you do this stuff when you want to finetune, maybe I am wrong.
About the rest, I guess I understood what you ment, but what about the train the text encoder concept training, my images on the concept images folder are only stuff like, "general image name" from 1 to 1000, I would have to name individually all these images too?

**Reply by nawnie** [reply]
no dont caption them as they get renamed when uploaded, they do not read the name only the text encoding of the image itself.

there are many ways to reserve your model but in general yes, low lr will help keep from over fitting.

lets use the tv show bugs bunny as an example.

for my instance image i want bugs bunny, elmer fudd, daffy duck, for sure. now if i want to generate more things in that style i have three options, 

1, i can caption images of cars food grass clouds people .... and add them to the instance images

2. i can hope the ai will pick it up via text encoder when training images that contain the subject.

3. add images not containing my subjects to concept images for the encoder to pick up, this seems to let the encoder know the things found in here should also be paid attention to when training the unet (this isnt fact just what ive seen happen in testing).

note that the text encoder ie CLIP knows looney tune characters if the image is clear and the subject is not obstructed, subjects that are like this will need less encoding, play with a clip encoder to find this out.

**Reply by leonnn1** [reply]
"no dont caption them as they get renamed when uploaded, they do not read the name only the text encoding of the image itself."

I think you are talking about the concept images here right? 

And when you say that  "things found in here should also be paid attention" 

you mean only visually that there is no metadata or name read on concept images, for them what matters is what is in the images, is that correct?
